src/main/java/net/mcreator/ui/minecraft/modellayers/JModelLayerList.java

```diff
@@ -21,72 +21,30 @@
 
 import net.mcreator.element.types.LivingEntity;
 import net.mcreator.ui.MCreator;
-import net.mcreator.ui.component.util.PanelUtils;
+import net.mcreator.ui.component.entries.JSimpleEntriesList;
 import net.mcreator.ui.help.IHelpContext;
 import net.mcreator.ui.init.L10N;
-import net.mcreator.ui.minecraft.JEntriesList;
 import net.mcreator.ui.validation.AggregatedValidationResult;
 
 import javax.swing.*;
 import java.awt.*;
-import java.util.ArrayList;
 import java.util.List;
-import java.util.Objects;
-import java.util.stream.Collectors;
 
-public class JModelLayerList extends JEntriesList {
-
-	private final List<JModelLayerListEntry> entryList = new ArrayList<>();
-
-	private final JPanel entries = new JPanel(new GridLayout(0, 1, 5, 5));
+public class JModelLayerList extends JSimpleEntriesList<JModelLayerListEntry, LivingEntity.ModelLayerEntry> {

```

Comment:
Given this feature along with #2007 defines what is called **states** for blocks/items, does it deserve to be moved to `net.mcreator.ui.minecraft.states.entity`?

Reply:
I would keep it here for now, we can refactor it later if deemed necessary